scripts/create-dev-stack.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+source ./scripts/common.sh
+
+# This script creates and configures a dev stack in the `pulumi` organization.
+#
+# Usage:
+#
+#   ./scripts/create-dev-stack.sh                         # Creates a new dev stack. Prompts for stack name.
+#   ./scripts/create-dev-stack.sh stackname               # Same as above, but uses the passed-in argument for the stack name.
+#   STACK_NAME=stackname ./scripts/create-dev-stack.sh    # Same as above, but uses the STACK_NAME environment variable instead.
+
+stack_name="${STACK_NAME:=$1}"
+
+if [ -z "$stack_name" ]; then
+    echo
+    echo "This script assumes you're logged into Pulumi Cloud and that you belong"
+    echo "to the 'pulumi' organization. The stack will be created in that organization"
+    echo "automatically, so you should omit the qualifying prefix 'pulumi/' from the name."
+    echo
+    read -p "What would you like to name this stack (e.g., 'christian')? " stack_name
+fi
+
+echo
+echo "Creating pulumi/${stack_name}..."
+echo
+
+pulumi -C infrastructure stack init "pulumi/${stack_name}"
+pulumi -C infrastructure config set aws:region us-west-2
+pulumi -C infrastructure config set certificateArn "arn:aws:acm:us-east-1:372027080554:certificate/54d0431c-2cf9-4c40-bd3c-324cfb8b7d32"
```

Comment:
I'm assuming the domain on this cert is for `*.pulumi-dev.io` in order for this to work? If it is, this is awesome and will be great for this!

Reply:
Yep, this assumed you're using that domain.